app/src/main/java/com/framgia/soundclound/screen/addtracktoalbum/AddTrackActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.framgia.soundclound.screen.addtracktoalbum;
+
+
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.framgia.soundclound.R;
+import com.framgia.soundclound.data.source.TrackRepository;
+import com.framgia.soundclound.data.source.remote.TrackRemoteDataSource;
```

Comment:
ERROR: Unused import: com.framgia.soundclound.data.source.remote.TrackRemoteDataSource. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports